### DISCUSSION TITLE
Why I can't use student pack

### DISCUSSION THREAD

**Discussion by likegravity** [community/community#112076] [GitHub Education] [answered]
![Screenshot_20240311_134318_Gmail.jpg](https://github.com/community/community/assets/110320707/06c23d70-ec07-4e3d-9714-e73532c9084e)

**Comment by moozunch** [comment, marked as answer]
Hi, I also got the same problem at first but this is how I do it: 

1. Go to settings > billing plan > payment infomation
![image](https://github.com/community/community/assets/112236945/959ba984-50dc-4b03-9e7a-88cec53a39e6)
2. Then fill your payment informartion, it's jut basic question you neeed to fill, you don't have to add your payment method
![image](https://github.com/community/community/assets/112236945/7fb79578-4c26-4919-b753-f3ff5f462b19)
3. then save, you might also have to reapply the submission, make sures your name are the same as your id card.

Try this as I also got this issues 
1. Activate the 2FA for your account

hope this help ^____^
